metrics/network/remote_network/remote-networking-test-common.sh

```diff
@@ -20,6 +20,7 @@
 
 SCRIPT_PATH=$(dirname "$(readlink -f "$0")")
 source "${SCRIPT_PATH}/../../lib/common.bash"
+LIB_DIR="${SCRIPT_PATH}/../../lib"
```

Comment:
I see that you are creating this `LIB_DIR` variable, but also I don't see where it is being used.

Reply:
@chavafg this is used for the save_results